fluent/client/ws_client.go

```diff
@@ -79,6 +80,8 @@ func (wcf *DefaultWSConnectionFactory) New() (ext.Conn, error) {
 		header.Add(AuthorizationHeader, wcf.AuthInfo.IAMToken())
 	}
 
+	dialer.TLSClientConfig = &tls.Config{InsecureSkipVerify: true} // nolint
```

Comment:
This shouldn't be set by default. Use the same pattern as here: https://github.com/IBM/fluent-forward-go/blob/main/fluent/client/conn_factory.go#L16

Reply:
There should be tests for this. See https://github.com/IBM/fluent-forward-go/blob/main/fluent/client/conn_factory_test.go#L118-L129 for inspiration